### PR TITLE
feat(omit): add parser omit

### PR DESCRIPTION
**Pull request by AMoreaux**
## What it does

- Add parser omit

### CODE REVIEW

**Review by ChucKN0risK — CHANGES_REQUESTED** [review]
Do you have a use case in mind?

**Comment by AMoreaux** [comment]
> Do you have a use case in mind?

Exclude `color` from textStyles.

**Review by ChucKN0risK — DISMISSED** [review]

File: parsers/omit/README.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+Learn more about how to configure Specify in the API documentation: [https://specifyapp.com/developers/cli](https://specifyapp.com/developers/cli)
+
+## Interface 
+```ts
+interface parser {
+  name: 'omit',
+  options?: {
+    keys: Array<string>
+    "filter"?: {
```
Comment (AllanMichay) [inline parsers/omit/README.md:15]: Why the `"` here?

File: parsers/omit/README.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+      "types": Array<string>
```
Comment (AllanMichay) [inline parsers/omit/README.md:16]: Why the `"` here?

File: parsers/omit/README.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+Array of object with at least a name's key
+
+```ts
+Array<{[key: string]: any}>
+```
+
+### Output
+
+Array of object with the keys given in options
```
Comment (AllanMichay) [inline parsers/omit/README.md:44]: Array of object without the given keys

**Review by AllanMichay — APPROVED** [review]

**Pull request merged**